src/ert/storage/local_ensemble.py

```diff
@@ -117,10 +119,14 @@ def _get_parameter(self, realization: int) -> bool:
             for parameter in self.experiment.parameter_configuration
         )
 
-    def _get_response(self, realization: int) -> bool:
+    def _get_response(self, realization: int, key: Optional[str] = None) -> bool:
```

Comment:
Not really important, but could we use a more descriptive name such as `_all_responses_exist_for_realization` or something like that?

Reply:
Agree